types/spotify-web-api-node/index.d.ts

```diff
@@ -563,13 +573,14 @@ declare class SpotifyWebApi {
     getMyTopTracks(options: GetTopOptions, callback: Callback<SpotifyApi.UsersTopTracksResponse>): void;
     getMyTopTracks(options?: GetTopOptions): Promise<Response<SpotifyApi.UsersTopTracksResponse>>;
 
-    /**
-     * Get the Current User's Recently Played Tracks
-     * @param options Options, being type, after, limit, before.
-     * @param callback Optional callback method to be called instead of the promise.
-     * @returns A promise that if successful, resolves into a paging object of tracks,
-     *          otherwise an error. Not returned if a callback is given.
-     */
+  /**
+   * Get the Current User's Recently Played Tracks
+   * @param options Options, being type, after, limit, before.
+   * @param callback Optional callback method to be called instead of the promise.
+   * @returns A promise that if successful, resolves into a paging object of play history objects,
+   *          otherwise an error. Not returned if a callback is given. Note that the response will be empty
+   *          in case the user has enabled private session.
+   */
```

Comment:
```suggestion
    /**
     * Get the Current User's Recently Played Tracks
     * @param options Options, being type, after, limit, before.
     * @param callback Optional callback method to be called instead of the promise.
     * @returns A promise that if successful, resolves into a paging object of play history objects,
     *          otherwise an error. Not returned if a callback is given. Note that the response will be empty
     *          in case the user has enabled private session.
     */
```